### ISSUE TITLE
Main CUL accessibility: Search node page input is missing label	

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#226]
See issue comments here:
https://culibrary.atlassian.net/browse/CULWEBDEVTEAM-1060

**Comment by mhk33** [comment]
It seems that a side effect of the update template.php was the disappearance of checkboxes in My Account. Jim removed that change and the checkboxes came back. So I reopened this issue.

**Comment by mhk33** [comment]
This is coming up as an error again in SiteImprove. See comments on this JIRA issue:
https://culibrary.atlassian.net/browse/CULWEBDEVTEAM-1060

**Comment by jgreidy** [comment]
I put a temporary solution in production. When the current version of the bootstrap theme is used, the Keywords: label goes away. The temporary solution just uses the previous version of bootstrap.

**Comment by mhk33** [comment]
This appears to be fixed now, it hasn't shown up as an issue for a while.
